src/main/java/pl/marekjava/PasswordManager.java

```diff
@@ -1,20 +1,32 @@
 package pl.marekjava;
 
+
+import javax.json.*;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
 import java.util.*;
 
 public class PasswordManager {
     private List<String> passwords = new ArrayList<>();
     private Map<String, Boolean> usedPasswords = new HashMap<>();
     private String currentPassword;
-    private List<Character> corectGuesses = new ArrayList<>();
+    private List<Character> correctGuesses = new ArrayList<>();
 
     public PasswordManager() {
-        passwords.add("Apetyt rośnie w miarę jedzenia");
-        passwords.add("Co dwie głowy, to nie jedna");
-        passwords.add("Cwiczenie czyni mistrza");
-        passwords.add("Darowanemu koniowi w zęby się nie zagląda");
-        passwords.add("Diabeł tkwi w szczegółach");
-        passwords.add("Elektryka prąd nie tyka");
+
+        JsonReader jsonReader = null;
+        try {
+            jsonReader = Json.createReader(new FileReader("src\\main\\resources\\passwords.json"));
```

Comment:
Takie podanie ścieżki wykrzaczy się gdy tylko a) ktoś uruchomi ten kod na innym systemie operacyjnym b) ktoś spakuje kod do jarka i będzie chciał odpalić aplikację z niego.
Żeby zaradzić problemowi a)najepiej użyć klasy `Path`, ma ona metodę `.of`, gdzie podaje się kolejne elementy ścieżki, a ona "skleja" je odpowiednio dla danego systemu operacyjnego. 
Co do b) [https://frontbackend.com/java/java-io-reading-a-file-from-the-resources](https://frontbackend.com/java/java-io-reading-a-file-from-the-resources)

Reply:
Ok. Zmieniłem zgodnie z punktem b. Dodatkowo dodałem warunek, że nie nalicza punktów, gdy litera już jest zgadnięta.